### PR TITLE
Arm action docs

### PR DESCRIPTION
**Pull request by yves-chan**
ARM Action docs

## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Documentation/Links/LinkActions.md [hunk 1]
```diff
@@ -51,6 +51,40 @@ When using the `Make this item a link` option, the following settings are availa
...
+|`Run button text from` | Label used on the run (execute) button to trigger the ARM action.|
+
+![Image showing ARM UX settings](../Images/ArmActionUXSettings.png)
+
+After these configurations are set, when the user clicks on the link, it will open up the view with the UX described in [ARM Action UX Settings](#ARM-Action-UX-Settings). From here, if the user clicks on the button specified by `Run button text from` it will run the ARM action using the values from [ARM Action UX Settings](#ARM-Action-UX-Settings). 
+
```
Comment (gardnerjr) [inline Documentation/Links/LinkActions.md:87]: is there an example of this?  would be useful to have a filled in version of the ux settings, and a corresponding version of the run blade that shows exactly what maches up where

should also indicate which of these values show up in notifications?

i don't think you mention notifications here at all, you probably should

File: Documentation/Links/LinkActions.md [hunk 2]
```diff
@@ -51,6 +51,40 @@ When using the `Make this item a link` option, the following settings are availa
...
+### ARM Action Settings Tab
+
+This section defines the ARM action API.
+
+| Source | Explanation |
+|:------------- |:-------------|
+|`ARM Action path` | The ARM action path. eg /subscriptions/:subscription/resourceGroups/:resourceGroup/someAction?api-version=:apiversion. Select from `Cell`, `Column`, `Static Value`, or `Parameter` in [Link sources](#link-sources).|
+|`Http Method` | Select an HTTP method. The available choices are `POST`, `PUT`, `PATCH`, `DELETE`|
+|`Long Operation` | Long Operations will poll the URI from the Azure-AsyncOperation response header from the original operation.|
```
Comment (gardnerjr) [inline Documentation/Links/LinkActions.md:66]: >Long Operation [](http://example.com/codeflow?start=2&length=14)

should probably be be a link for this too, so people know what this means?  both here and possibly in the infoballoon in the ui?

File: schema/workbook.json [hunk 3]
```diff
@@ -192,10 +192,6 @@
                         "title": {
                             "$ref": "#/definitions/title",
                             "description": "title displayed above the grid/chart"
-                        },
-                        "exportParameters": {
-                            "type": "boolean",
-                            "description": "indicates if the parameters defined inside this group are visible outside the group"
                         }
```
Comment (gardnerjr) [inline schema/workbook.json:195]: why did this get removed?

File: schema/workbook.json [hunk 4]
```diff
@@ -1420,6 +1447,83 @@
...
+            "$id": "#/definitions/armActionContext",
+            "type": "object",
+            "properties": {
+                "pathSource": {
+                    "type": "string",
+                    "description": "source of the ARM path",
+                    "enum": [
+                        "static", "column", "parameter"
+                    ]
+                },
```
Comment (gardnerjr) [inline schema/workbook.json:1461]: didn't you remove this "source" stuff?

File: scripts/extractStrings.js [hunk 5]
```diff
@@ -15,7 +15,8 @@ const LocKeys = [
   "defaultItemsText",
   "loadButtonText",
   "noDataMessage",
-  "markDown" // specific to cohorts
+  "markDown", // specific to cohorts
+  "actionName"
```
Comment (gardnerjr) [inline scripts/extractStrings.js:19]: >actionName [](http://example.com/codeflow?start=3&length=10)

why actionname?  isn't this `runLabel` and `actionMessage` and `resultMessage` and things?

File: Documentation/Links/LinkActions.md [hunk 6]
```diff
@@ -51,6 +51,40 @@ When using the `Make this item a link` option, the following settings are availa
...
+
+Use this setting to invoke an ARM action by specifying the ARM api details.
+
+### ARM Action Settings Tab
+
+This section defines the ARM action API.
+
+| Source | Explanation |
+|:------------- |:-------------|
+|`ARM Action path` | The ARM action path. eg /subscriptions/:subscription/resourceGroups/:resourceGroup/someAction?api-version=:apiversion. Select from `Cell`, `Column`, `Static Value`, or `Parameter` in [Link sources](#link-sources).|
```
Comment (gardnerjr) [inline Documentation/Links/LinkActions.md:64]: > Select from `Cell`, `Column`, `Static Value`, or `Parameter` in [Link sources](#link-sources) [](http://example.com/codeflow?start=139&length=94)

this all still has the link sources stuff, doesn't apply does it?

File: Documentation/Links/LinkActions.md (hunk 1 above)
Comment (yves-chan) [reply Documentation/Links/LinkActions.md:91]: Provided an example from the settings to the end result, and specified the setting that shows up in notification. Also I added a screenshot about notifications after the button is clicked

File: schema/workbook.json (hunk 3 above)
Comment (yves-chan) [reply schema/workbook.json:195]: I think this is in the wrong section, it's already in the groups definition. This was removed from the metrics definition

File: scripts/extractStrings.js (hunk 5 above)
Comment (yves-chan) [reply scripts/extractStrings.js]: should be actionName and runLabel

**Review by gardnerjr — APPROVED** [review]
:shipit:

**Pull request merged**